onnxruntime/core/providers/cpu/tensor/scatter_nd.cc

```diff
@@ -131,7 +131,7 @@ Status ScatterND::Compute(OpKernelContext* context) const {
 }
 
 Status ScatterND::ScatterNumber(const Prepare& p) const {
-#ifdef USE_OPENMP
+#ifdef _OPENMP
```

Comment:
I guess changes to use ParallelFor will come in a separate PR?

Reply:
Yes